package.json

```diff
@@ -66,15 +91,43 @@
     ],
     "keybindings": [
       {
-        "key": "alt+c",
-        "command": "clay.showTopLevelForm"
+        "key": "ctrl+alt+a f",
+        "command": "clay.makeFile"
       },
       {
-        "key": "alt+shift+c",
-        "command": "clay.makeFile"
+        "key": "ctrl+alt+a q f",
+        "command": "clay.makeFileQuarto"
+      },
+      {
+        "key": "ctrl+alt+a r f",
+        "command": "clay.makeFileRevealJs"
+      },
+      {
+        "key": "ctrl+alt+a c",
+        "command": "clay.makeCurrentForm"
+      },
+      {
+        "key": "ctrl+alt+a q c",
+        "command": "clay.makeCurrentFormQuarto"
+      },
+      {
+        "key": "ctrl+alt+a a",
+        "command": "clay.makeTopLevelForm"
+      },
+      {
+        "key": "ctrl+alt+a ctrl+alt+a",
+        "command": "clay.makeTopLevelForm"
+      },
+      {
+        "key": "ctrl+alt+a q a",
+        "command": "clay.makeTopLevelFormQuarto"
+      },
+      {
+        "key": "ctrl+alt+a b",
+        "command": "clay.browse"
       },
       {
-        "key": "alt+shift+w",
+        "key": "ctrl+alt+a w",
```

Comment:
I don't think this warrants a keybinding. And if it is some other less hammered on command, I think we should remove those bindings as well.

Reply:
Makes sense to me.